project-base/storefront/pages/wishlist.tsx

```diff
@@ -35,7 +36,21 @@ const WishlistPage: NextPage<ServerSidePropsType> = () => {
 export const getServerSideProps = getServerSidePropsWrapper(
     ({ redisClient, domainConfig, t }) =>
         async (context) =>
-            initServerSideProps({ context, redisClient, domainConfig, t }),
+            initServerSideProps({
+                context,
+                redisClient,
+                domainConfig,
+                t,
+                prefetchedQueries: [
+                    {
```

Comment:
When I refresh the wishlist page, I get the following error

```
Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.

Warning: Expected server HTML to contain a matching <div> in <div>.

See more info here: https://nextjs.org/docs/messages/react-hydration-error
```

Reply:
I wasn't able to reproduce.